app/components/view_component_reflex/component.rb

```diff
@@ -214,8 +214,7 @@ def unsafe_instance_variables
         :@view_context, :@lookup_context, :@view_renderer, :@view_flow,
         :@virtual_path, :@variant, :@current_template, :@output_buffer, :@key,
         :@helpers, :@controller, :@request, :@tag_builder, :@state_initialized,
-        :@_content_evaluated, :@_render_in_block, :@__vc_controller, :@__vc_helpers,
-        :@__cached_content, :@__vc_variant, :@__vc_content_evaluated, :@__vc_render_in_block,
+        :@_content_evaluated, :@_render_in_block, :@__cached_content,
```

Comment:
I didn't remove these values since they may still exist in older versions of view component, but I believe they should have been renamed to be consistent with the `__vc` prefix other internal ivars have.